src/server/render.js

```diff
@@ -375,6 +378,9 @@ export function createRenderFunction (
   isUnaryTag: Function,
   cache: any
 ) {
+  const options = baseOptions
+  options.isUnaryTag = isUnaryTag || baseOptions.isUnaryTag
+  compile = createCompiler(options)
```

Comment:
This would cause each `createRenderer` to overwrite the compiler from the previous call. (i.e. creating a second renderer with different options will affect a previously created renderer)